packages/app-builder-lib/src/macPackager.ts

```diff
@@ -450,7 +450,7 @@ export default class MacPackager extends PlatformPackager<MacConfiguration> {
     await BluebirdPromise.map(readdir(packContext.appOutDir), async (file: string): Promise<any> => {
```

Comment:
If you change the return of this larger function to `Promise<boolean>`, intellisense will still flag that this function isn't returning the correct values. I'll take a look on refactoring this function entirely